packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -284,6 +306,7 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 		this.editor.exitTransaction();
 
```

Comment:
Any reason not to just emit the event here, right after exiting the transaction? (same for abortTransaction)

Reply:
No reason, that's a good point. I've updated the code do emit it here.